### PR TITLE
Added pull request template

### PR DESCRIPTION
**Pull request by mserranom**
Minimal pull request template, including a checkbox for contributor license agreement, mostly for us to double check that the contributor has signed before merging.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Contributor Agreement
+
+- [ ] I confirm I have signed the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)
```
Comment (NyanKiyoshi) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: There is nothing to sign, probably "read" instead?

```suggestion
- [ ] I confirm I have read the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)
```

Comment (mserranom) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: If you follow it there's a link to a form where you can sign the agreement.

Comment (NyanKiyoshi) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Oh saw it now, thanks!

**Comment by mserranom** [comment]
Moved to the bottom.

**Pull request merged**